src/ssl.c

```diff
@@ -17590,6 +17590,11 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
             BIO_set_retry_read(wr);
         }
     }
+
+    int wolfSSL_BIO_method_type(const WOLFSSL_BIO *b)
```

Comment:
I've moved all the wolfSSL_BIO functions to bio.c.
Move it there too. Thanks!

Reply:
Moved to `bio.c`.